README.md

```diff
@@ -55,17 +55,23 @@ If you'd like to help with any of these, feel free to submit a PR or ask how you
 
 The table of projects which are _currently_ supported.
 
+<!-- AUTO-GENERATED-CONTENT:START (PROJECTS:path=./data/projects.json) -->
```

Comment:
@DSchau what are your thoughts on adding some sort of `WARNING` comment here?

Reply:
Updated!